.travis.yml

```diff
@@ -1,15 +1,15 @@
+dist: focal
 language: node_js
-
+node_js:
+  - node
+  - lts/*
 branches:
   only:
-    - gh-pages
-
-cache:
-  directories:
-    - node_modules
-
+    - master
```

Comment:
👍 can we make this `main` instead?

Reply:
I'd rather not. `master` is the historical standard. I see GH changed its default to `main` for new repos, but in Git it's still `master`.